commands/plugin_marketplace_e2e_test.go

```diff
@@ -141,3 +141,31 @@ func removePluginIfInstalled(s *MmctlE2ETestSuite, pluginID string) {
 		s.Require().Contains(appErr.Error(), "Plugin is not installed.")
 	}
 }
+
+func (s *MmctlE2ETestSuite) TestPluginMarketplaceListCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("List Marketplace Plugins for Admin User", func(c client.Client) {
+		printer.Clean()
+
+		err := pluginMarketplaceListCmdF(c, &cobra.Command{}, nil)
+
+		pluginList := printer.GetLines()
+
+		// This checks whether there is an output from the command - returned list can be of length >= 0
+		s.Require().Len(pluginList, len(pluginList))
+		s.Require().Nil(err)
```

Comment:
```suggestion
		s.Require().NoError(err)
```

Reply:
@streamer45  pushed updated code with the suggested changes